README.md

```diff
@@ -38,6 +43,54 @@ If you need support, please either raise an issue on Github or ask on [Slack](ht
 
 Substra warmly welcomes any contribution. Feel free to fork the repo and create a pull request.
 
+## How to test
+
+Install `substrafl` in editable mode with developper dependencies.
+In addition, install `substra` and `substra-tools` in editable mode.
+It is recommended to install all the libraries in a Python virtual env.
+
+```sh
+git clone git@github.com:Substra/substrafl.git
+pip install -e "substrafl[dev]"
+git clone git@github.com:Substra/substra.git
+pip install -e substra
+git clone git@github.com:Substra/substra-tools.git
+pip install -e "substra-tools"
+```
+
+Now you can use the following command from `subtrafl` top level directory to run tests:
+
+```sh
+cd substrafl
+make test-subprocess
+```
+
+### Running advanced test suites
+
+Substra can be used in three different modes: using Python subprocesses (`subprocess`),
+ using Docker (`docker`) and using Kubernetes (`remote`).
+
+The command `make test-subprocess` runs the test suite in subprocess mode. It's lightweight and perfect to start.
+
+To test with the Docker mode, you will need Docker installed and running on your machine.
+If necessary, install it using [Docker Desktop](https://www.docker.com/products/docker-desktop/).
+
+The following command runs the test suites in subprocess and Docker mode:
+
+```sh
+make test-local
+``
+
+Please be warned that some of these tests are slow and the whole test suite might require a couple hours to complete.
+
+To try out a local deployment with Kubernetes, please follow the [installation instructions](https://docs.substra.org/en/stable/contributing/local-deployment.html) provided in the doc.
```

Comment:
Updated the URL following https://github.com/Substra/substra-documentation/pull/257/